.github/ISSUE_TEMPLATE/suggestion.yml

```
@@ -0,0 +1,44 @@
+name: Word Suggestion
+description: Share a word suggestion
+title: ":blue_book: "
+labels: ["Type · Definitions", "Status · Unconfirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out word definition request!
+  - type: input
+    id: word
+    attributes:
+      label: What word would you like to see defined?
+      description: What word do you think need to be in the Self-Defined dictionary and why?
+    validations:
+      required: true
+  - type: textarea
+    id: related-resources
+    attributes:
+      label: Please share any relevant links
+      description: Please provide at least one related example or definition.
+    validations:
+      required: true
+  - type: textarea
+    id: definition
+    attributes:
+      label: If you already have a definition in mind, share it here
+      description: If you have an idea about what the definition should be, please share it here.
+      render: shell
+  - type: checkboxes
+    id: assign-to-me
+    attributes:
+      label: Would you like to take ownership of defining this word?
```

```suggestion
      label: Would you like to lead/steward defining this word?
```
Not sure of the best word here, but "take ownership" might read intimidatingly and individualistically. I also would love to prime them that definitions should be crafted collaboratively, with sensitivity readers and editors.